templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  <p>Are you sure to delete the manufacturer: {{ manufacturer }}</p>
+  <p>
+  <li> All cars of this manufacturer will be also deleted!</li>
+  </p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger"/>

```

Comment:
Add `Cancel` button apart from `Delete` one as mentioned in the checklist. Update corresponding screenshots